transmart-core-api/src/main/groovy/org/transmartproject/core/tree/TreeResource.groovy

```diff
@@ -0,0 +1,34 @@
+package org.transmartproject.core.tree
+
+import org.transmartproject.core.users.User
+
+interface TreeResource {
+
+    /**
+     * Finds tree nodes with prefix $rootKey up to depth $depth lower than the $rootKey level.
```

Comment:
It would be nice to mention that tree could vary from user to user and why.

Reply:
Agree. I will extend the documentation.